src/types.cairo

```diff
@@ -45,3 +45,17 @@ pub struct MarketReserveData {
     liquidation_bonus: felt252,
     debt_limit: felt252
 }
+
+#[derive(Copy, Drop, Serde)]
+pub struct Config {
```

Comment:
After a part of methods from `IAirdrop` is removed this one gonna be unneeded too.